app/Views/public/templates/whatsapp.php

```diff
@@ -0,0 +1,8 @@
+<?php $this->section('js'); ?>
+<script src="assets/js/public/whatsapp.js"></script>
+<?php $this->endSection(); ?>
+<button class="btn-whatsapp" id="btn-whatsapp">
```

Comment:
@CarlosMay7 creo que seria mas facil si esto fuera un anchor con un href a la direccion que pusiste en el javascript no?

algo asi 

```suggestion
<a target="_blank" href="https://api.whatsapp.com/send?phone=.....">
```